quickfixj-codegenerator/src/main/java/org/quickfixj/codegenerator/MessageCodeGenerator.java

```diff
@@ -447,6 +477,7 @@ public static void main(String[] args) {
                 task.setMessagePackage("quickfix." + version.toLowerCase());
                 task.setOutputBaseDirectory(new File(args[2]));
                 task.setFieldPackage("quickfix.field");
+                task.setUtcTimestampPrecision(task.utcTimestampPrecision);
```

Comment:
Not sure how often this method is used, but this does not seem correct.

Reply:
Hi @the-thing. Can you please elaborate? I did not understand what is wrong here.

Reply:
Ah, I see now that this looks stupid. Sets the value to it self... I'm not familiar with the usage og this main method, so maybe one of you could suggest a solution?

Reply:
I believe this methods is used when you want to run code generation locally from IDE etc.. I would follow the same pattern as e.g. overwrite/orderedFields/useDecimal. Create an option, get an option or default value, set it.

Reply:
OK. Thank you. I will have a look at it.

Reply:
Suggestion in e1b5bee92c8aeb4893e5f5a77936499dc78a7a71. Created the `getOption(String key, String defaultValue)` in case someone want to change the default value to something other than null later. I will check back to see if you have any feedback tomorrow.